aria-practices.html

```diff
@@ -2772,8 +2772,8 @@ <h3>Tabs</h3>
       <section class="notoc">
         <h4>Examples</h4>
         <ul>
-          <li><a href="examples/tabs/tabs-1/tabs.html">Tabs With Automatic Activation</a>: A tabs widget where tabs are automatically activated and their panel is displayed when they receive focus.</li>
-          <li><a href="examples/tabs/tabs-2/tabs.html">Tabs With Manual Activation</a>: A tabs widget where users activate a tab and display its panel by pressing <kbd>Space</kbd> or <kbd>Enter</kbd>.</li>
+          <li><a href="examples/tabs/tabs-automatic.html">Tabs With Automatic Activation</a>: A tabs widget where tabs are automatically activated and their panel is displayed when they receive focus.</li>
```

Comment:
This is a lot clearer, nice!

Reply:
Two main issues:
1.  I am not sure it is a a good practice for the APG to reference contemporary musicians, it might be misunderstood as some type of endorsement for those musicians or at least an unfair improvement to their SEO, so I did not feel comfortable using them in the updates I made. 
2. We will be adding at least one more example that will add and delete tabs and this was easier for me to find additional related content for those examples.

Reply:
Should we also update the carousel example to remove all references to the TV shows?